django_rq/worker_pool.py

```diff
@@ -0,0 +1,38 @@
+import django
+from multiprocessing import Process, get_start_method
+from typing import Any
+
+from rq.worker_pool import WorkerPool, run_worker
+
+
+class DjangoWorkerPool(WorkerPool):
+    def get_worker_process(
+        self,
+        name: str,
+        burst: bool,
+        _sleep: float = 0,
+        logging_level: str = "INFO",
+    ) -> Process:
+        """Returns the worker process"""
+        return Process(
+            target=run_django_worker,
+            args=(name, self._queue_names, self._connection_class, self._pool_class, self._pool_kwargs),
+            kwargs={
+                '_sleep': _sleep,
+                'burst': burst,
+                'logging_level': logging_level,
+                'worker_class': self.worker_class,
+                'job_class': self.job_class,
+                'serializer': self.serializer,
+            },
+            name=f'Worker {name} (WorkerPool {self.name})',
+        )
+
+
+def run_django_worker(*args: Any, **kwargs: Any) -> None:
+    # multiprocessing library default process start method may be
+    # `spawn` or `fork` depending on the host OS
+    if get_start_method() == 'spawn':
+        django.setup()
+
+    run_worker(*args, **kwargs)
```

Comment:
Similar to @jackkinsella's comment, we need to use our own `run_worker` command that calls `reset_db_connections` before running the original `run_worker` command: https://github.com/rq/django-rq/blob/025a53f379e48a9d5483421260d8052f143169bf/django_rq/utils.py#L150